### ISSUE TITLE
Add instructions on how to `Verify this session`

### ISSUE THREAD

**Issue by xobs** [vector-im/element-web#13975]
**Is your suggestion related to a problem? Please describe.**
Riot now creates a popup asking users to verify the session. This appears every time Riot is started. There is no clear method on how to actually accomplish this.

**Describe the solution you'd like**
This window pops up after clicking "Verify":

![image](https://user-images.githubusercontent.com/238325/84232147-47333f00-ab22-11ea-93d9-32d3ca36ec81.png)

However, this window doesn't say what to actually to in order to accomplish verification. It seems to imply that you can find more information by using one of those clients, however I am running the same version of Riot on another machine, and I'm running RiotX on Android, and neither of those has any additional information.

**Describe alternatives you've considered**
Unknown.

**Additional context**
This appeared in a recent update to Riot.

**Comment by matthijskooijman** [comment]
I was also confused by this same popup. Initially, I thought that it said that maybe it meant that all of my other clients were too old and I was supposed to upgrade them, which was weird, since Riot Web is always upgraded and I was using latest RiotX.

There was no clear "next" step from this dialog. The "Use recovery passphrase or key" option is styled in a way that suggests it is a fallback option (which I think is true) and "Skip" is obviously not the happy path either.

From reading https://blog.riot.im/e2e-encryption-by-default-cross-signing-is-here/ I have learned that the idea is that you go to another, already logged, client and start verifying the new session from there. This fact could probably be made more clear, and with more specific instructions on what to do exactly in the other clients (or at least where the start of the UI path to verify is located).

Also, this screen now shows all supported clients. If technically possible, it might be better to limit this to showing any clients actually used by the user (maybe show a list of logged in devices, possibly filtered by recent activity) so any instructions can be more specific to the current user.

**Comment by zilti** [comment]
> From reading https://blog.riot.im/e2e-encryption-by-default-cross-signing-is-here/ I have learned that the idea is that you go to another, already logged, client and start verifying the new session from there

Yea, I came to the same conclusion. But the problem is, there is no way to do that in the Android app.

**Comment by xobs** [comment]
Yes, I gather now that RiotX should have popped up a dialog box to do verification, but that never happened.

**Comment by zilti** [comment]
It is really weird too because apparently, even the "old" Riot app (which I was still on) seems to support it. And in one case it actually *did* pop up a dialog, but only once.

**Comment by tribut** [comment]
Some concrete ideas how to improve this are in #14415

**Comment by RichardJECooke** [comment]
Hi. New Element user. Can someone please explain how to 'verify this login'??  Just downloaded the app and this is the first thing I googled. Can't see anything to click on in the web browser.

**Comment by RichardJECooke** [comment]
And why do I need to do this anyway. The app doesn't explain it

**Comment by LuckySandwich** [comment]
I am equally as confused by this.

On android, it says "use an existing session to verify this one".. all it gives me is a Skip button and a red arrow.  Clicking the red arrow forces me to either cancel or return to where I just was (and there's nothing else to do on that screen)

Very confusing how this is set up

**Comment by JambonBeurreVodka** [comment]
I can't even "verify this session" on my android when i log in on my desktop. How about you don't push new features until they work?

**Comment by Qbsoon** [comment]
I don't even have "Use recovery passphrase or key" button. I logged in on my browser on pc and on app on my android device. Still can't verify these two sessions.

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/vector-im/element-meta/issues/1870
